### PR TITLE
Added PlayerLecternPageChangeEvent

### PR DESCRIPTION
**Pull request by Machine-Maker**

### CODE REVIEW

File: Spigot-Server-Patches/0602-Added-PlayerLecternPageChangeEvent.patch [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+             switch (i) {
+                 case 1:
+                     j = this.containerProperties.getProperty(0);
+-                    this.a(0, j - 1);
++                    // Paper start
++                    playerLecternPageChangeEvent = new PlayerLecternPageChangeEvent((org.bukkit.entity.Player) entityhuman.getBukkitEntity(), ((CraftInventoryLectern) getBukkitView().getTopInventory()).getHolder(), PlayerLecternPageChangeEvent.PageChangeDirection.LEFT, j, j - 1);
++                    if (!playerLecternPageChangeEvent.callEvent()) {
++                        return false;
++                    }
++                    this.a(0, playerLecternPageChangeEvent.getNewPage());
```
Comment (MiniDigger) [inline Spigot-Server-Patches/0602-Added-PlayerLecternPageChangeEvent.patch]: can we get a obf helper here?

**Review by MiniDigger — DISMISSED** [review]
looks mine, just the obf helper I guess

File: Spigot-Server-Patches/0602-Added-PlayerLecternPageChangeEvent.patch (hunk 1 above)
Comment (Machine-Maker) [reply Spigot-Server-Patches/0602-Added-PlayerLecternPageChangeEvent.patch]: Done

**Review by MiniDigger — DISMISSED** [review]

**Review by zachbr — DISMISSED** [review]

**Comment by Proximyst** [comment]
@limbo-app ready

**Review by limbo-app — APPROVED** [review]
Pull request automatically approved for merging - a core team member has previously approved this pull request.

**Pull request merged**